docs_nnx/mnist_tutorial.md

```diff
@@ -246,6 +246,8 @@ plt.show()
 Create a `jit`-compiled model inference function (with `nnx.jit`) - `pred_step` - to generate predictions on the test set using the learned model parameters. This will enable you to visualize test images alongside their predicted labels for a qualitative assessment of model performance.
 
 ```{code-cell} ipython3
+model.eval() # Switch to evaluation mode.
```

Comment:
Consider explaining a bit more why this is necessary.

Reply:
Added an explanation.